testing/integration/serverless/logs_ingestion_test.go

```diff
@@ -20,6 +20,6 @@ func TestLogIngestionFleetManaged(t *testing.T) {
 		Local: false,
 		Sudo:  true,
 	})
-
+	t.Skip("Skip serverless tests until we can enroll an agent with 9.3.0-SNAPSHOT.")
```

Comment:
👍  Let's remember to get a revert PR for this line up as soon as this PR here is merged.